### PR TITLE
device consistency

### PR DESCRIPTION
**Pull request by deepchatterjeeligo**
refactoring device consistency from #122. This moves the project, psd estimator, and waveform generator to the right device.

### CODE REVIEW

File: projects/train/train/data/waveforms/generator/cbc.py [hunk 1]
```diff
@@ -103,8 +103,13 @@ def time_domain_strain(self, **parameters):
                 the right of the waveform in seconds
         """
 
-        # TODO: support time domain waveforms
-        hc, hp = self.waveform(self.frequencies[self.freq_mask], **parameters)
+        device = parameters["chirp_mass"].device
+        freqs = torch.clone(self.frequencies).to(device)
```
Comment (ravioli1369) [inline projects/train/train/data/waveforms/generator/cbc.py:107]: What's the reason for cloning the frequency grid here? Does passing it into the approximant change it?

Comment (deepchatterjeeligo) [reply projects/train/train/data/waveforms/generator/cbc.py:107]: The `self.frequency` can be on the cpu. So I clone it and transfer to the gpu for use in this function, without impacting the global attribute.

**Comment by deepchatterjeeligo** [comment]
anything else on this?

**Comment by ravioli1369** [comment]
nope

**Pull request merged**